pkg/helpers/newapp/newapptest/newapp_test.go

```diff
@@ -1015,14 +1015,14 @@ func TestNewAppRunBuilds(t *testing.T) {
 			name: "successful build from dockerfile",
 			config: &cmd.AppConfig{
 				GenerationInputs: cmd.GenerationInputs{
-					Dockerfile: "FROM openshift/origin:v1.0.6\nUSER foo",
+					Dockerfile: "FROM centos:centos8\nUSER foo",
```

Comment:
Why not use UBI 9? We know it works.

```suggestion
					Dockerfile: "FROM registry.access.redhat.com/ubi9/ubi-micro:latest\nUSER foo",
```

Reply:
This doesn't work because it requires a different tag instead of `latest`